packages/server/src/app/routes/v2/channels.ts

```diff
@@ -10,17 +11,35 @@ export const BASE_URL = `/api/v2/channels`;
 
 const router = new Router();
 
+function sendViaFirebase(ctx) {
+  // Executed after all other middleware
+  const status = ctx.status;
+  if (status === 200 || status === 201) {
+    const to = ctx.body.to;
+    const messagePayload = ctx.body.messagePayload;
+    if (to && messagePayload) {
+      send(to, messagePayload);
+    }
+  }
+}
+
 router.post(`${BASE_URL}`, koaBody(), async ctx => {
   const { queue } = ctx.request.body;
   if (queue === 'GAME_ENGINE') {
-    return await handleGameRequest(ctx);
+    const response = await handleGameRequest(ctx);
+    sendViaFirebase(response);
+    return response;
   } else {
     const action = ctx.request.body;
 
     if (await isNewProcessAction(action)) {
-      return await handleNewProcessAction(ctx);
+      const response = await handleNewProcessAction(ctx);
+      sendViaFirebase(response);
+      return response;
```

Comment:
If we are sending commitments via firebase, do we need to return the commitments in the http request? Should we instead return something that's useful to the firebase function?

Reply:
Firebase relay removes the message from the realtime database after forwarding the message to the hub. So any information from the hub that signals whether to remove the message is useful. No need to reply with a commitment. A response code is sufficient.